### PR TITLE
Portfolio: setup

### PR DESCRIPTION
**Pull request by Hisoka37**
#  Set up Portfolio project 
In this PR I : 
- Add HTML header to follow HTML semantics.
- Link Logo with the homepage. 
-  wrap icons list to `<ul>` tag with `<li>`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <div>
+      <nav class="navbar">
+        <img src="/assets/My Logo.svg" class="logo" />
+        <img class="menu" src="assets/Side.svg" alt="menu" />
+      </nav>
```
Comment (MeqdamAlqudah) [inline index.html]: - [x] Please the container of the Toolbar section should be a `header` tag as it is semantically better while the hamburger button only should be wrapped in a nav tag.

- [x] Please use the present tense in the Pull request description.
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <div>
+      <nav class="navbar">
+        <img src="/assets/My Logo.svg" class="logo" />
```
Comment (MeqdamAlqudah) [inline index.html]: - [x] Please the logo should be a link to the home page, if you check websites like youtube and facebook you will notice that their logo sends the user to the home page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <img class="menu" src="assets/Side.svg" alt="menu" />
+      </nav>
+      <section class="my-section">
+        <div>
```
Comment (MeqdamAlqudah) [inline index.html:20]: - [x] Please your code should have margins (top, bottom, left, right) defined in the design, it is a really important skill to give the same design requested from you.

| Your code  | Figma|
| ------------- | ------------- |
| image | image|

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        </div>
+        <div class="contact-me">
+          <h2 class="connect">LET'S CONNECT</h2>
+        </div>
+        <div class="buttons">
+          <a href=""><img src="/assets/Github.svg" alt="Github" /></a>
+          <a href=""><img src="/assets/Twitter.svg" alt="Twitter" /></a>
+          <a href=""><img src="/assets/medium.svg" alt="Medium" /></a>
+          <a href=""><img src="/assets/Vector.svg" alt="" /></a>
+        </div>
```
Comment (MeqdamAlqudah) [inline index.html]: - [x] Please these icons form a list and should be wrapped in a `ul` tag with `li` elements this is semantically better.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+            experience! If you like what you see and <br />
+            have a project you need coded, don’t <br />
+            hestiate to contact me.
+          </p>
+        </div>
+        <div class="contact-me">
+          <h2 class="connect">LET'S CONNECT</h2>
+        </div>
+        <div class="buttons">
+          <a href=""><img src="/assets/Github.svg" alt="Github" /></a>
```
Comment (MeqdamAlqudah) [inline index.html]: - [x] Please Let's connect should be a link or a button and it should have the same font family as in the design 
image

- [x] Please Headline section should  has the margins (top, bottom, left, right) defined in the design

| Your code  | Figma code |
| ------------- | ------------- |
| image  | image  |

- [x] Please use the same font color as in the design .
your design 
image
Figma 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Hisoka37 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights

:+1: Readme file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Hisoka37  👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**